### PR TITLE
E2E: Updated README

### PR DESCRIPTION
**Pull request by ljelinkova**

### CODE REVIEW

File: ee_tests/README.md [hunk 1]
```diff
@@ -5,30 +5,30 @@
...
 6. verifies deployed application in stage and run environments
-7. verifies data in deployments page and analytics report
+7. verifies data on the dashboard page and analytics report
+
+### Other Suites ###
+The `che` suite tests more advanced scenario where the user opens Che, updates codebase and the Openshift.io
+picks up the changes and runs the build number 2.
 
-It is expected to be expanded to do something meaningful in Che and to interact with Planner in the future.
+The `logintest` is the simplest test where user only logs in and logs our. It runs pretty frequently on Jenkins on all clusters.
```
Comment (ppitonak) [inline ee_tests/README.md]: a typo... `logs our`

File: ee_tests/README.md [hunk 2]
```diff
@@ -5,30 +5,30 @@
...
 
 1. logs into the platform
 2. creates a new space
-3. creates a new Vert.x project
+3. creates a new project
 4. opens Che workspace
 5. verifies that pipeline behaves correctly
 6. verifies deployed application in stage and run environments
-7. verifies data in deployments page and analytics report
+7. verifies data on the dashboard page and analytics report
```
Comment (ppitonak) [inline ee_tests/README.md]: Do we want to mention that it also creates a work item in beta config?

File: ee_tests/README.md [hunk 3]
```diff
@@ -60,7 +64,9 @@ npm run image:build
 ```
 npm run image:start
 ```
-
+#### Review the test results ####
+All artifacts are copied from docker image to `target/screenshots` direcotry. See detailed description
```
Comment (ppitonak) [inline ee_tests/README.md]: replace `docker image` with `docker container`

File: ee_tests/README.md (hunk 2 above)
Comment (ljelinkova) [reply ee_tests/README.md]: OK

**Pull request merged**